### DISCUSSION TITLE
Unable to connect N8N with SupaBase self hosted on Elestio (Docker)

### DISCUSSION THREAD

**Discussion by PrasannaPtools** [supabase/supabase#30699] [Questions]
![image](https://github.com/user-attachments/assets/b28c2625-6137-445d-98a0-2f6ca787a448)

Whenever I try to connect to Supabase on N8N I get "Unauthorized" as a response. In the "Service Role Secret" field I've used the Anon Key, Service Role Key & the Service Role Secret that generated the two previously mentioned and I always get the same "Unauthorized" response.

Is there something I'm missing? Or anyway to debug this?

**Comment by neutrinotek** [comment]
You ever get this figured out? I'm having the same issue. Feel like I've tried everything.

**Comment by PattrnData** [comment]
you get this resolved?

**Comment by PrasannaPtools** [comment]
Unfortunately, I wasn't able to figure this with a self hosted instance of Supabase so I ended up using it on their cloud and that seems to work for me.

**Comment by Audiojoy72** [comment]
but that could not be a final solution sending your personal / private informations to a external database somewhere out there. i m trying to get supabase to work since around 3 weeks daily 5-8h with the programming help of openai, claude and gemini. Me and my AI friends are goin from one to another problem while trying to get a local vector store for n8n to run. will describe that in another topic. supabase seems to me the undefeatable boss level of all setups.

**Comment by Joaquin-Urruti** [comment]
I'm having the same issue with both instances hosted on elestio...
